### PR TITLE
 Fix GET, POST Fields resourse (#165)

### LINKED ISSUE #165: Fix GET, POST Fields resourse

### PR DESCRIPTION
**Pull request by AndriiKret**

### CODE REVIEW

**Review by Howrran — DISMISSED** [review]

**Review by yurdosii — DISMISSED** [review]

**Review by Howrran — DISMISSED** [review]

File: src/app/routers/field.py [hunk 1]
```diff
@@ -133,21 +130,25 @@ def post(self):
...
+                to_row=setting_autocomplete.get('toRow')
             )
 
-        if response is None:
+        if field is None:
             raise BadRequest("Could not create")
 
-        return Response(status=201)
+        field_json = FieldService.to_json(field)
+        print(f'field_json: {field_json}')
```
Comment (IvanKostiv) [inline src/app/routers/field.py]: delete print

File: src/app/services/field.py [hunk 2]
```diff
@@ -269,6 +269,9 @@ def create_text_or_number_field(  # pylint: disable=too-many-arguments
             raise FieldAlreadyExist()
         data = FieldNumberTextSchema().dump(field)
 
+        if is_strict is True:
```
Comment (IvanKostiv) [inline src/app/services/field.py]: if strict

File: src/app/routers/field.py (hunk 1 above)
Comment (AndriiKret) [reply src/app/routers/field.py]: Fixed

**Review by Howrran — APPROVED** [review]

File: src/app/services/field.py (hunk 2 above)
Comment (AndriiKret) [reply src/app/services/field.py]: Fixed

**Review by IvanKostiv — APPROVED** [review]

**Review by zzeropilot — APPROVED** [review]

**Pull request merged**